src/components/Calculator.js

```diff
@@ -21,41 +21,47 @@ class Calculator extends Component {
   render() {
     const { total, next, operation } = this.state;
     return (
+      <div>
+        <div className="app1">
 
-      <div className="app1">
-        <div className="calculator">
-          <div className="display">
-            { total}
-            {operation}
-            {next}
-
-          </div>
-          <div className="operators">
-            <button onClick={this.clickEvent} name="÷" type="button" color="primary" value="">&#247;</button>
-            <button onClick={this.clickEvent} name="x" type="button" color="primary" value="">&times;</button>
-            <button onClick={this.clickEvent} name="-" type="button" color="primary" value="">-</button>
-            <button onClick={this.clickEvent} name="+" type="button" color="primary" value="">+</button>
-            <button onClick={this.clickEvent} name="=" type="button" color="primary" value="">=</button>
+          <div className="header-calc">
+            <h1>Welcome to our page</h1>
+            {/* eslint-disable-next-line max-len */}
+            <p> Nullam vulputate magna ac accumsan tristique. Vivamus aliquam, ipsum at facilisis suscipit, arcu augue bibendum ex, a commodo purus eros eu arcu. Suspendisse et placerat erat. Cras lectus nisi, molestie at auctor non, vulputate non est. Suspendisse sodales molestie congue. Nulla metus sem, egestas non porta id, lobortis in nunc. Vivamus cursus pulvinar sem, in consequat lacus dignissim non. Fusce imperdiet eleifend lacus vitae placerat. Phasellus imperdiet sed ex quis imperdiet. Integer et cursus mauris. Integer commodo in justo in gravida.</p>
```

Comment:
- [ ] Good job so far 👍 It would be great if you could follow the wireframe closely, as a result kindly remove this heading and paragraph and apply the correct text as given on the template 💯 
- Your design:
![Annotation 2022-06-02 065714](https://user-images.githubusercontent.com/85113567/171563226-c8d92d37-7529-47aa-99fd-d87f90be3067.jpg)
- Wireframe:
![Annotation 2022-06-02 065905](https://user-images.githubusercontent.com/85113567/171563255-d6bf7e0a-8c55-4a1f-b741-e47caf0f4ed9.jpg)